src/components/Annotation/ReplyForm/ReplyForm.tsx

```diff
@@ -45,67 +47,58 @@ export const ReplyForm = (props: ReplyFormProps) => {
   useEffect(() => {
     if (textarea.current && props.autofocus)
       setTimeout(() => textarea.current?.focus({ preventScroll: true }), 1);
-  }, [props.autofocus])
+  }, [props.autofocus]);
 
   const onSubmit = (evt?: React.MouseEvent) => {
     evt?.preventDefault();
 
-    if (textarea.current?.value) {      
+    if (value) {
       const body: AnnotationBody = {
         id: uuidv4(),
         annotation: props.annotation.id,
-        creator: {  
+        creator: {
           id: me.id,
           name: me.name,
-          avatar: me.avatar
+          avatar: me.avatar,
         },
         created: new Date(),
         purpose: 'commenting',
-        value: textarea.current.value
+        value: JSON.stringify(value),
+        render_type: 'quill',
```

Comment:
Hm, this feels like a bit of a mixed approach. If we implement it like this, that means we'd never have a plaintext mode anymore. (Every reply would be hard-wired to rich text, if I understand correctly.)

I wonder what the purpose then would be for even keeping the dual support (editable text area and rich text). IMO we should decide between two approaches:
- Allow both rich text and plaintext. In this case, the ReplyForm would have to support both modes, too.
- Decide to go for Rich Text only. In this case, there would be no need for switching between components. The only thing we'd need to do is to check if the body value is plaintext or JSON, and crosswalk from plaintext to Quill format, to maintain backwards compatibility with older annotations.

Reply:
I considered an inline upgrade to rich text but was not sure.  If you agree I can probably do an inline translation of all text on render and a conversion on edit.

Reply:
I think the question is whether we want the ReplyField to support plaintext at all. If so, we'd need another switching mechanism. Not sure how that would work though. Would have to be a config argument, or env variable passed from Astro. Essentially to tell the client "render a plaintext reply field by default" vs. "render the RTE by default". 

Really not sure what to do here, and no strong opinions. Some thoughts: (also CC-ing @jamiefolsom.)
- I believe it's really a matter of deciding whether we want to be able to switch the RTE feature on and off for specific clients in the future, or just give it to anyone. (My gut feeling is that we should just make it the default. Definitely keeps the code simpler)
- Did Bonn say they want rich text, too? If so, that might be our answer.

Caveat: Quill introduces a bit of a difficulty for the use case of __exporting annotations from Recogito__. When exporting to CSV (or W3C Web Anno) there's no simple way for us to produce readable output. Therefore, I wonder: 

- is there a way for us to cross-walk Quill content to plaintext on the server, during the export?
- if not, should we store two sets of values in the annotation? The Quill/JSON value __and__ and a plaintext version? (I'm assuming there is an easy way to pull an unformatted copy of the text from Quill - which may or may not be the case...)

Reply:
Yes.  Quill has a [getText](https://quilljs.com/docs/api/#gettext) method which converts the quill content to text, omitting non-text content.  You would just instantiate quill in your CSV or TEI export method and convert all of the quill bodies.

Reply:
It looks like quill requires access to the `document`, i.e. it won't work on the server. The Recogito data exports are just API endpoints, which means we can't crosswalk easily.

Unless we want to store two copies of the comment (quill and plaintext), we have two options:

- Build our own crosswalk, by figuring out how the Quill JSON format works and then stripping/concatenating the tokens ourselves
- Adding a client-side crosswalk to the text exporter.
  - The user would click the download button
  - There would be JS on the client that fetches the annotations, instantiates Quill, runs the conversion client side
  - After the conversion is done, there would be a [fake download link](https://stackoverflow.com/questions/19721439/download-json-object-as-a-file-from-browser) which would provide the same experience as an actual download from an API endpoint.

However: while technically possible, I'd rather not use the client-side hack! One of the first things that people wanted to do in old Recogito was to harvest data from the APIs. Once the exports rely on browser JS to provide the right data, we are preventing any kind of script interaction.

Therefore: I think that storing both versions or 'reverse-engineering' our own Quill crosswalk might be our only options.